action.yml

```diff
@@ -37,7 +37,27 @@ outputs:
     description: >
       The JSON-serialized Sigstore bundle containing the signed provenance
       statement and related verification material.
+    value: ${{ steps.translate-output.outputs.bundle }}
 
 runs:
-  using: node20
-  main: dist/index.js
+  using: 'composite'
+  steps:
+    - uses: actions/attest-build-provenance@v1
+      id: attest
+      with:
+        subject-path: ${{ inputs.subject-path }}
+        subject-digest: ${{ inputs.subject-digest }}
+        subject-name: ${{ inputs.subject-name }}
+        push-to-registry: ${{ inputs.push-to-registry }}
+        github-token: ${{ inputs.github-token }}
+    - id: translate-output
+      if: runner.os != 'Windows'
+      shell: bash
+      run: |
+        read -r line < ${{ steps.attest.outputs.bundle-path }}
+        echo "bundle=$line" >> $GITHUB_OUTPUT
```

Comment:
The primary difference between the early-access action and the public action is the way the action output is handled:

*  In the early-access action, the JSON-serialized bundle is passed back in the `bundle` output variable.
* In the new action, any attestation bundles are written to a file and the path to that file is passed back in the `bundle-path` output variable.

This bit translates the new scheme to the old one for backward compatibility. We don't handle Windows cause there's not easy way to make this work on both Windows and non-Windows runners.

Reply:
The only user I'd worry about for the Windows platform is Homebrew... but I believe yesterday they switched to `actions/attest-build-provenance` so I think we can proceed!

Reply:
Also, I checked and they are *not* using the action output